docker/Dockerfile

```diff
@@ -14,7 +14,8 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
     zlib1g \
     libbz2-dev \
     liblzma-dev \
-    libreadline-dev
+    libreadline-dev \
+    libglpk40
```

Comment:
This is to address the individual Dockerfile problem noted on #312

Reply:
Cam we put this in a separate PR? I think it deserves its own testing.